odoo/modules/module.py

```diff
@@ -141,6 +141,16 @@ def initialize_sys_path():
     if base_path not in ad_paths:
         ad_paths.append(base_path)
 
+    # add odoo.addons.__path__
+    try:
+        for ad in __import__('odoo.addons').addons.__path__:
+            ad = os.path.abspath(ad)
+            if ad not in ad_paths:
+                ad_paths.append(ad)
+    except ImportError:
+        # no distribution provides odoo.addons
+        pass
```

Comment:
@sle-odoo the try/except is not necessary anymore since Odoo always provides `odoo.addons`.